cabal-install/Distribution/Client/Dependency/Types.hs

```diff
@@ -166,7 +168,11 @@ showPackageConstraint (PackageConstraintStanzas pn ss) =
 -- It is not specified if preferences on some packages are more important than
```

Comment:
It would be good to update this comment and explain the meaning of multiple ranges.